src/components/MainMap.vue

```diff
@@ -24,9 +26,16 @@ export default {
       center: latLng(50.06465, 19.94498),
       url: "http://{s}.tile.osm.org/{z}/{x}/{y}.png",
       attribution:
-        '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors',
-      markerpos: latLng(50.025276, 19.939498)
+        '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
```

Comment:
Maybe these properties should be moved to separate file, like map.config.js?

Reply:
As far as I'm aware, "zoom" and "center" are dynamic, and using an outside config file wouldn't work, because modules.exports is read-only.

In this case, isn't using a config file for two constants that are not likely to change in the near future a bit excessive?

Reply:
You're right. 

In case of expanding map configuration or adding separate map instance to the app, this could be reconsidered.